data/fixtures/recorded/surroundingPair/changePair.yml

```diff
@@ -0,0 +1,25 @@
+languageId: typescriptreact
+command:
+  version: 7
+  spokenForm: change pair
+  action:
+    name: clearAndSetSelection
+    target:
+      type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: surroundingPair, delimiter: any}
+  usePrePhraseSnapshot: false
+initialState:
+  documentContents: |
+    <div bbb={() => null}>hello &</div>
```

Comment:
This test fails on main. `&` is not a valid syntax in jsx. Because of this `>` in  `=>` was incorrectly used as a pair delimiter.